### PR TITLE
Delete ISM330DLC IMU driver

### PR DESCRIPTION
**Pull request by bkueng**
We don't use it anymore as it does not perform as expected. Thus no need to keep maintaining it.

### CODE REVIEW

File: boards/px4/fmu-v6x/init/rc.board_sensors [hunk 1]
```diff
@@ -2,29 +2,28 @@
...
+icm20649 -R 2 -s start
 
-# Possible internal compass
+# Internal magnetometer on I2c
 bmm150 -I start
 
 # Possible internal Baro
-bmp388 -I start
 bmp388 -I -a 0x77 start
+bmp388 -I start
```
Comment (davids5) [inline boards/px4/fmu-v6x/init/rc.board_sensors:26]: Should we be explicit with the address?

**Review by davids5 — APPROVED** [review]
See inline comment.

**Review by dagar — APPROVED** [review]
Good riddance.

**Comment by CarlOlsson** [comment]
@bkueng May I ask, what didn't work as expected?

**Comment by bresch** [comment]
@CarlOlsson During some vibration tests we found out that the accelerometers were not performing as well as the other usual IMUs (ICM, MPU, ...). It especially showed a large bias change at several frequencies.

**Pull request merged**